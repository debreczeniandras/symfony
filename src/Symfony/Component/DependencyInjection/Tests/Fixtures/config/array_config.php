<?php

use Symfony\Component\DependencyInjection\Tests\Fixtures\Bar;

return [
    'parameters' => [
        'foo' => 'bar',
    ],
    'services' => [
        '_defaults' => [
            'public' => true,
        ],
        Bar::class => null,
        'my_service' => [
            'class' => Bar::class,
            'arguments' => ['%foo%'],
        ],
    ],
];
