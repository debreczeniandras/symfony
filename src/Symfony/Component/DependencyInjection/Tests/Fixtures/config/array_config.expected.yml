parameters:
    foo: bar

services:
    service_container:
        class: Symfony\Component\DependencyInjection\ContainerInterface
        public: true
        synthetic: true
    Symfony\Component\DependencyInjection\Tests\Fixtures\Bar:
        class: Symfony\Component\DependencyInjection\Tests\Fixtures\Bar
        public: true
    my_service:
        class: Symfony\Component\DependencyInjection\Tests\Fixtures\Bar
        public: true
        arguments: [bar]
